schemas/groups/electrical.json

```diff
@@ -238,263 +242,427 @@
           "units": "W"
         }
       }
+    },
+  
+    "chargerQuantities": {
+      "type": "object",
+      "title": "Charger Quantities",
+      "description": "Common charger quantities",
+      "properties": {
+        "allOf": [{
+            "$ref": "#/definitions/identity"
+            }],
+        "chargingAlgorithm": {
+          "type": "object",
+          "description": "Algorithm being used by the charger",
+          "allOf": [
+            {
+              "$ref": "../definitions.json#/definitions/commonValueFields"
+            },
+            {
+              "properties": {
+                "value": {
+                  "type": "string",
+                  "enum": [
+                    "trickle",
+                    "two stage",
+                    "three stage",
+                    "four stage",       
+                    "constant current",
+                    "constant voltage",
+                    "custom profile"
+                  ]
+                }
+              }
+            }
+          ]
+        },
```

Comment:
I know this has been a long battle, but please bear with me. I want to get this thing merged!

Not being well versed in anything but very simple charging setups the `chargerMode` and `mode` are not intuitively obvious. 

The mode enum values, all but _unknown_ and _other_, are all prefixed _charging_, so I was about to suggest that the property be named `chargeMode` and drop the prefixes. Furthermore enum values with spaces are look unwieldy to me. 

But then I discovered _chargerMode_ above. This property really needs better documentation. This is about multiple charger setups? Should the property name carry that word?